src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -1,21 +1,45 @@
 import { ICSSInJSStyle } from '../../../../../types/theme'
+import { IChatMessageProps } from '../../../../components/Chat/ChatMessage'
+import { IChatMessageVariables } from './chatMessageVariables'
+import { pxToRem } from '../../../../lib'
 
 const chatMessageStyles = {
-  root: ({ props, variables }): ICSSInJSStyle => ({
+  root: ({
+    props: p,
+    variables: v,
+  }: {
+    props: IChatMessageProps
+    variables: IChatMessageVariables
+  }): ICSSInJSStyle => ({
+    display: 'flex',
     position: 'relative',
-    padding: '1rem',
     marginTop: '1rem',
     marginBottom: '1rem',
-    ...(props.mine
+    ...(p.mine
```

Comment:
Levi always prefers destructuring objects at the beginning of a function.

Reply:
well it was not destructured before and doing so will increase code size; Levi prefers shorter code as well; I think destructuring should happen once we use more props there or logic becomes more complex